pkg/schema/generate.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
I'd say we should move it under `cmd/`, as we don't have `main` package in `pkg/` (aside from things that get have build tags and only used with `go generate`, IIRC).